flujo/cli/architect_command.py

```
@@ -0,0 +1,1159 @@
+    goal: Optional[str],
+    name: Optional[str],
+    output_dir: Optional[str],
+    non_interactive: bool,
+    pattern: Optional[str],
+    iterable_name: Optional[str],
+    reduce_mode: Optional[str],
+    conversation: Optional[bool],
+    stop_when: Optional[bool],
+    propagation: Optional[str],
+    body_steps: Optional[str],
+    map_step_name: Optional[str],
```

Ruff (0.14.5)

Unused function argument: `map_step_name`

(ARG001)